scripts/hubot-netrunnerdb-cards.coffee

```diff
@@ -314,3 +344,31 @@ module.exports = (robot) ->
 				cardString += " + " + randomCard.title
 
 		res.send cardString
+
+	robot.hear /^!find (.*)/, (res) ->
+		conditions = []
+		for part in res.match[1].toLowerCase().split(" ")
+			if out = part.match(/([a-z]+)(:|=|<|>|!)([\w]+)/)
```

Comment:
Yeah, good catch. I think originally I was going to allow longer keys (e.g. "influence") but I decided to copy NRDB one-char codes in the end. Will update.